db/connection.js

```diff
@@ -0,0 +1 @@
+var mongoose = require require("mongoose")
```

Comment:
This exercise is from the first day.  We didn't expect you to use mongoose.  I'm glad to see you didn't use mongoose after this initial start (with syntax errors).